scripts/west_commands/runners/openocd.py

```diff
@@ -98,6 +100,8 @@ def do_add_parser(cls, parser):
                             help='openocd telnet port, defaults to 4444')
         parser.add_argument('--gdb-port', default=DEFAULT_OPENOCD_GDB_PORT,
                             help='openocd gdb port, defaults to 3333')
+        parser.add_argument('--no-halt', default=True, action='store_true',
```

Comment:
I think `default=True, ` should be removed.

If you have `action='store_true'`, the default is automatically set to `False`, which looks like what you want to have instead.

Reply:
Thanks for the hint. Fixed